packages/gatsby/src/utils/generic-on-error.ts

```diff
@@ -0,0 +1,6 @@
+import reporter from "gatsby-cli/lib/reporter"
+
+export function genericOnError(_ctx, event): void {
+  const error = event.data
+  reporter.panicOnBuild(error)
```

Comment:
(It would need an optional chain here though)

Reply:
Done